### ISSUE TITLE
Missing channel

### ISSUE THREAD

**Issue by tim222tim** [lightningnetwork/lnd#996]
### Background

One of three channels disappears from "listchannels" and "channelbalance". 

### Your environment

lnd 0.4
Ubuntu 16
bitcoind 16

### Steps to reproduce

I opened three mainnet channels and made a few payments. Later, the second of the three channels was missing from "listchannels" and the balance from that channel was not reflected in "channelbalance". Oddly, the channel point was still listed in "freereport" under channel fees.  I stopped lnd and restarted it, and then the missing channel came back. After a few minutes, the same channel disappeared again.

### Expected behaviour

Channel should persist.

### Actual behaviour

Channel is disappearing on it's own.

**Comment by Roasbeef** [comment]
What do you mean "disappeared"? If a channel is in the process of being opened/closed, it'll catually be returned in the `pendingchannels` call not `listchannels`.

**Comment by tim222tim** [comment]
Ah. Thanks, man. I think the only issue is my lack of understanding. It didn't occur to me that a channel that I hadn't attempted to close might be in the process of closing. Makes sense now.

Here's what I think happened: One of my channels wasn't shown in my channel balance or list of channels. The other party must have been closing the channel, so it was in my pending channels. I didn't think to look in pending channels. 

When I stopped lnd and started it again, is it possible that there was a brief time when the channel in question was in my channel list before the process discovered that it was pending and moved it to the pending list? That's what I think I saw, and why I thought it appeared and then disappeared. 

If anything, perhaps it would be good to see a total balance somewhere, as in wallet + channel + pending. (Maybe that's already available?) That would have made things more clear for me.

Thanks again for all the great work.

-- Tim
